[Bug][DataSource] upgrade from 2.0.0 to 2.0.5 then oracle datasource will not to work

### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar issues.


### What happened

upgrade from 2.0.0 to 2.0.5 then oracle datasource will not to work:

[ERROR] 2022-03-15 09:38:30.560 TaskLogLogger-class org.apache.dolphinscheduler.plugin.task.sql.SqlTask:[160] - sql task error: java.lang.RuntimeException: JDBC connect failed
[ERROR] 2022-03-15 09:38:30.561 org.apache.dolphinscheduler.server.worker.runner.TaskExecuteThread:[207] - task scheduler failure
java.lang.RuntimeException: JDBC connect failed
        at org.apache.dolphinscheduler.plugin.datasource.api.client.CommonDataSourceClient.checkClient(CommonDataSourceClient.java:106)
        at org.apache.dolphinscheduler.plugin.datasource.api.client.CommonDataSourceClient.<init>(CommonDataSourceClient.java:55)
        at org.apache.dolphinscheduler.plugin.datasource.oracle.OracleDataSourceClient.<init>(OracleDataSourceClient.java:27)
        at org.apache.dolphinscheduler.plugin.datasource.oracle.OracleDataSourceChannel.createDataSourceClient(OracleDataSourceChannel.java:29)
        at org.apache.dolphinscheduler.plugin.datasource.api.plugin.DataSourceClientProvider.lambda$getConnection$0(DataSourceClientProvider.java:64)
        at java.util.concurrent.ConcurrentHashMap.computeIfAbsent(ConcurrentHashMap.java:1660)
        at org.apache.dolphinscheduler.plugin.datasource.api.plugin.DataSourceClientProvider.getConnection(DataSourceClientProvider.java:58)
        at org.apache.dolphinscheduler.plugin.task.sql.SqlTask.executeFuncAndSql(SqlTask.java:183)
        at org.apache.dolphinscheduler.plugin.task.sql.SqlTask.handle(SqlTask.java:154)
        at org.apache.dolphinscheduler.server.worker.runner.TaskExecuteThread.run(TaskExecuteThread.java:189)
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
        at com.google.common.util.concurrent.TrustedListenableFutureTask$TrustedFutureInterruptibleTask.runInterruptibly(TrustedListenableFutureTask.java:125)
        at com.google.common.util.concurrent.InterruptibleTask.run(InterruptibleTask.java:57)
        at com.google.common.util.concurrent.TrustedListenableFutureTask.run(TrustedListenableFutureTask.java:78)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
        at java.lang.Thread.run(Thread.java:748)
Caused by: org.springframework.jdbc.CannotGetJdbcConnectionException: Failed to obtain JDBC Connection; nested exception is java.sql.SQLSyntaxErrorException: ORA-00923: FROM keyword not found where expected

        at org.springframework.jdbc.datasource.DataSourceUtils.getConnection(DataSourceUtils.java:83)
        at org.springframework.jdbc.core.JdbcTemplate.execute(JdbcTemplate.java:376)
        at org.springframework.jdbc.core.JdbcTemplate.execute(JdbcTemplate.java:431)
        at org.apache.dolphinscheduler.plugin.datasource.api.client.CommonDataSourceClient.checkClient(CommonDataSourceClient.java:104)
        ... 16 common frames omitted
Caused by: java.sql.SQLSyntaxErrorException: ORA-00923: FROM keyword not found where expected

### What you expected to happen

all my job faild

### How to reproduce

a jdbc datasouce create in 2.0.0 or before then upgrade to 2.0.5 

### Anything else

_No response_

### Version

2.0.5

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

>Caused by: org.springframework.jdbc.CannotGetJdbcConnectionException: Failed to obtain JDBC Connection; nested exception is java.sql.SQLSyntaxErrorException: ORA-00923: FROM keyword not found where expected

Hi @lordk911 can you provide the sql in `how to reproduce`?

> > Caused by: org.springframework.jdbc.CannotGetJdbcConnectionException: Failed to obtain JDBC Connection; nested exception is java.sql.SQLSyntaxErrorException: ORA-00923: FROM keyword not found where expected
> 
> Hi @lordk911 can you provide the sql in `how to reproduce`?

it's because the datasource create in 2.0.0 didn't not have property : validationQuery

--- a/dolphinscheduler-datasource-plugin/dolphinscheduler-datasource-oracle/src/main/java/org/apache/dolphinscheduler/plugin/datasource/oracle/OracleDataSourceClient.java
+++ b/dolphinscheduler-datasource-plugin/dolphinscheduler-datasource-oracle/src/main/java/org/apache/dolphinscheduler/plugin/datasource/oracle/OracleDataSourceClient.java
@@ -20,11 +20,16 @@ package org.apache.dolphinscheduler.plugin.datasource.oracle;
 import org.apache.dolphinscheduler.plugin.datasource.api.client.CommonDataSourceClient;
 import org.apache.dolphinscheduler.spi.datasource.BaseConnectionParam;
 import org.apache.dolphinscheduler.spi.enums.DbType;
+import org.apache.dolphinscheduler.spi.utils.Constants;
 
 public class OracleDataSourceClient extends CommonDataSourceClient {
 
     public OracleDataSourceClient(BaseConnectionParam baseConnectionParam, DbType dbType) {
         super(baseConnectionParam, dbType);
+    }
+
+    protected void setDefaultValidationQuery(BaseConnectionParam baseConnectionParam) {
+        baseConnectionParam.setValidationQuery(Constants.ORACLE_VALIDATION_QUERY);
     }
 
 }

